test/health_test.go

```diff
@@ -249,63 +247,53 @@ func TestRuntimeMetrics(t *testing.T) {
-		"test-gateway.production.all-workers.runtime.mem.stack.mcacheInuse",
-		"test-gateway.production.all-workers.runtime.mem.stack.mcacheSys",
-		"test-gateway.production.all-workers.runtime.mem.stack.mspanInuse",
-		"test-gateway.production.all-workers.runtime.mem.stack.mspanSys",
-		"test-gateway.production.all-workers.runtime.mem.stack.sys",
-		"test-gateway.production.all-workers.runtime.mem.sys",
-		"test-gateway.production.all-workers.runtime.mem.total",
+		"test-gateway.production.per-worker.runtime.cpu.cgoCalls",
+		"test-gateway.production.per-worker.runtime.cpu.count",
+		"test-gateway.production.per-worker.runtime.cpu.goMaxProcs",
+		"test-gateway.production.per-worker.runtime.cpu.goroutines",
+		"test-gateway.production.per-worker.runtime.mem.alloc",
+		"test-gateway.production.per-worker.runtime.mem.frees",
+		"test-gateway.production.per-worker.runtime.mem.gc.count",
+		"test-gateway.production.per-worker.runtime.mem.gc.cpuFraction",
+		"test-gateway.production.per-worker.runtime.mem.gc.last",
+		"test-gateway.production.per-worker.runtime.mem.gc.next",
+		"test-gateway.production.per-worker.runtime.mem.gc.pause",
+		"test-gateway.production.per-worker.runtime.mem.gc.pauseTotal",
+		"test-gateway.production.per-worker.runtime.mem.gc.sys",
+		"test-gateway.production.per-worker.runtime.mem.heap.alloc",
+		"test-gateway.production.per-worker.runtime.mem.heap.idle",
+		"test-gateway.production.per-worker.runtime.mem.heap.inuse",
+		"test-gateway.production.per-worker.runtime.mem.heap.objects",
+		"test-gateway.production.per-worker.runtime.mem.heap.released",
+		"test-gateway.production.per-worker.runtime.mem.heap.sys",
+		"test-gateway.production.per-worker.runtime.mem.lookups",
+		"test-gateway.production.per-worker.runtime.mem.malloc",
+		"test-gateway.production.per-worker.runtime.mem.otherSys",
+		"test-gateway.production.per-worker.runtime.mem.stack.inuse",
+		"test-gateway.production.per-worker.runtime.mem.stack.mcacheInuse",
+		"test-gateway.production.per-worker.runtime.mem.stack.mcacheSys",
+		"test-gateway.production.per-worker.runtime.mem.stack.mspanInuse",
+		"test-gateway.production.per-worker.runtime.mem.stack.mspanSys",
+		"test-gateway.production.per-worker.runtime.mem.stack.sys",
+		"test-gateway.production.per-worker.runtime.mem.sys",
+		"test-gateway.production.per-worker.runtime.mem.total",
 	}
 
 	for i, m := range metrics {
 		assert.Equal(t, testData[i], m.Name, "expected correct name")
 
-		tags := m.Tags
-		assert.Equal(t, 3, len(tags), "expected 3 tags")
-		expectedTags := map[string]string{
-			"env":     "test",
-			"host":    getHostname(),
-			"service": "test-gateway",
-		}
-		for tag := range tags {
-			assert.Equal(t, expectedTags[tag.GetTagName()], tag.GetTagValue(), "expected tag value to be correct")
-		}
-	}
-}
-
-func getHostname() string {
-	host, err := os.Hostname()
-	if err != nil {
-		return "unknown"
+		// TODO: Why are common tags not emitted?
```

Comment:
if you dont think common tags work then lets remove all common tag related configuration and create our own default tags.